### ISSUE TITLE
Force return_format=CSV

### ISSUE THREAD

**Issue by Rafnuss** [davidclarance/rabm#17]
JSON is actually not currently supported. From what I understand, it doesn't bring any advantages compared to CSV. Consequently, I would remove it from the function arguments and documentation

Which do you prefer?

```
# return_format only support CSV
  return_format <- 'CSV'
```

or

`"http://api.adu.org.za/sabap2/v2/R/{start_date}/{end_date}/{region_type}/{region_id}/{return_type}/species/{species_id}?format=CSV"`

**Comment by Rafnuss** [comment]
I am in favor of the second option. Already implemented for `get_species_list`

**Comment by davidclarance** [comment]
Yeah agree, the second solution makes the most sense. Thanks for implementing. Will implement in all the other functions once I do a clean up.
